<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Tobor 1.0</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
          <rect>
            <x>10</x>
            <y>10</y>
            <width>1180</width>
            <height>700</height>
          </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <widget class="QWidget" name="widget" native="true">
              <widget class="QGraphicsView" name="graphicsView">
                <property name="geometry">
                  <rect>
                    <x>5</x>
                    <y>5</y>
                    <width>1180</width>
                    <height>700</height>
                  </rect>
                </property>
              </widget>
            </widget>
          </item>
        </layout>
      </widget>
    </widget>
    <widget class="QMenuBar" name="menubar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>826</width>
          <height>22</height>
        </rect>
      </property>
      <widget class="QMenu" name="menuFile">
        <property name="title">
          <string>File</string>
        </property>
        <addaction name="actionshowSVG"/>
      </widget>
      <widget class="QMenu" name="menuGame">
        <property name="title">
          <string>Game</string>
        </property>
        <addaction name="actionNewGame"/>
        <addaction name="actionStopGame"/>
      </widget>
      <widget class="QMenu" name="menuEdit">
        <property name="title">
          <string>Edit</string>
        </property>
      </widget>
      <widget class="QMenu" name="menuHelp">
        <property name="title">
          <string>Help</string>
        </property>
        <addaction name="actionAbout"/>
      </widget>
      <addaction name="menuFile"/>
      <addaction name="menuGame"/>
      <addaction name="menuEdit"/>
      <addaction name="menuHelp"/>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
    <action name="actionshowSVG">
      <property name="text">
        <string>showSVG</string>
      </property>
    </action>
    <action name="actionNewGame">
      <property name="text">
        <string>New Game</string>
      </property>
      <property name="enabled">
        <bool>true</bool>
      </property>
    </action>
    <action name="actionStopGame">
      <property name="text">
        <string>Stop Game</string>
      </property>
      <property name="enabled">
        <bool>false</bool>
      </property>
    </action>
    <action name="actionAbout">
      <property name="text">
        <string>About</string>
      </property>
    </action>
  </widget>
  <resources/>
  <connections/>
</ui>
